e2e/ansible/playbooks/resiliency/test-node-eviction-out-of-disk/test-node-eviction-out-of-disk.yml

```diff
@@ -100,7 +116,7 @@
            timeout: 120
 
        - name:  Get node on which application pod is scheduled
-         shell: source ~/.profile; kubectl get pods -n {{ namespace }} -o wide | grep percona | awk {'print $7'}
+         shell: source ~/.profile; kubectl get pods -n {{ namespace }} -o wide -l name=percona --no-headers | awk {'print $7'}
```

Comment:
`kubectl get pods -n {{ namespace }} -o wide -l name=percona --no-headers -o custom-columns=:spec.nodeName`

(since we are making some change here, might as well put some checks we like :) )

Reply:
ok @ksatchit